core/src/main/scala/org/apache/spark/rdd/ZippedPartitionsRDD.scala

```diff
@@ -45,7 +45,8 @@ private[spark] class ZippedPartitionsPartition(
 private[spark] abstract class ZippedPartitionsBaseRDD[V: ClassTag](
     sc: SparkContext,
     var rdds: Seq[RDD[_]],
-    preservesPartitioning: Boolean = false)
+    preservesPartitioning: Boolean = false,
```

Comment:
can you rename this preservesPartitioner, and add some classdoc to explain the parameters?